libs/core/src/lib/object-marker/object-marker.component.ts

```diff
@@ -13,14 +13,15 @@ import { CssClassBuilder } from '@fundamental-ngx/core/utils';
 @Component({
     // tslint:disable-next-line:component-selector
     selector: '[fd-object-marker]',
-    template: ` <i class="fd-object-marker__icon" [ngClass]="' sap-icon--' + glyph" *ngIf="glyph"></i>
+    template: ` <i class="fd-object-marker__icon" [ngClass]="' sap-icon--' + glyph" *ngIf="glyph" aria-hidden="true"></i>
```

Comment:
Why do you add `aria-hidden="true"`?

Reply:
I have added this to avoid reading of label two times as it contains both icon and label. I will remove this now as its not contain any label within icon. This changes were brought from another PR as part of combining both core and platform change fix for object marker.

Reply:
I have removed this now.